packages/webapp/src/components/Modals/CropCatalogSpotlightModal/index.js

```diff
@@ -4,16 +4,18 @@ import React from 'react';
 import { Label } from '../../Typography';
 import { useTranslation } from 'react-i18next';
 
-export default function CropCatalogSpotLightModal({ dismissModal }) {
+export default function CropCatalogSpotlightModal({ dismissModal }) {
   const { t } = useTranslation();
 
   return (
     <ModalComponent
       dismissModal={dismissModal}
-      title={'Crop Catalog'}
+      title={t('CROP_CATALOGUE.CROP_CATALOG')}
       buttonGroup={
         <>
-          <Button sm>Next</Button>
+          <Button onClick={dismissModal} sm>
+            {t('CROP_CATALOGUE.NEXT')}
```

Comment:
use common:NEXT for more commonly used translations. Can check for them in common.json